### PR TITLE
Fix PSA pin statuses when unpinned

### PR DESCRIPTION
**Pull request by flea89**
Ticket

### CODE REVIEW

File: packages/api/test/pin.spec.js [hunk 1]
```diff
@@ -667,6 +667,16 @@ describe('Pinning APIs endpoints', () => {
       assert.strictEqual(getEffectivePinStatus(pins), 'queued')
     })
 
+    it('should return "queued" at least 1 pin has Unpinned status', () => {
```
Comment (alexandrastoica) [inline packages/api/test/pin.spec.js:670]: Did you mean to say "IF at least one pin...?"

Comment (flea89) [reply packages/api/test/pin.spec.js]: good catch!

File: packages/api/src/pins.js [hunk 2]
```diff
@@ -51,8 +51,12 @@ export const getEffectivePinStatus = (pins) => {
     return 'pinning'
   }
 
+  // The cluster returns either Unpinned as "initial status",
+  // Remote when it has tracked the cid or PinQueued when  it has queued it for pinning.
```
Comment (vasco-santos) [inline packages/api/src/pins.js:55]:
```suggestion
  // The cluster returns as an initial status either "Unpinned",
  // "Remote" when it has tracked the cid or "PinQueued" when  it has queued it for pinning.
```

Comment (vasco-santos) [reply packages/api/src/pins.js:55]: nit: I think it reads more clear this way

File: packages/api/src/pins.js [hunk 3]
```diff
@@ -51,8 +51,12 @@ export const getEffectivePinStatus = (pins) => {
...
+  // All this statuses are translate to "queued" in respect to PSA statuses.
```
Comment (vasco-santos) [inline packages/api/src/pins.js:56]:
```suggestion
  // All this statuses are translated to "queued" in respect to PSA statuses.
```

File: packages/api/src/pins.js (hunk 2 above)
Comment (alanshaw) [reply packages/api/src/pins.js:55]: Remote means another node in the cluster is pinning it.

Comment (alanshaw) [reply packages/api/src/pins.js:55]: I think anything after `Queued` - at that point it has been allocated and all the nodes know where it should be pinned.

File: packages/api/src/pins.js (hunk 3 above)
Comment (alanshaw) [inline packages/api/src/pins.js:56]:
```suggestion
  // All these statuses are translated to "queued" in respect to PSA statuses.
```

**Review by alanshaw — APPROVED** [review]

File: packages/api/src/pins.js (hunk 2 above)
Comment (flea89) [reply packages/api/src/pins.js]: I've updated the whole block, hope it is clearer now.

**Pull request merged**